management/virtualmachine/resourceextensions_test.go

```diff
@@ -0,0 +1,38 @@
+package virtualmachine
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/management/testutils"
+)
+
+func init() {
+	flag.BoolVar(&extraVerbose, "resourceextensions.verbose", false, "List all resource extensions")

```

Comment:
I'd like to have an extra level of verbosity. Don't want to pollute the output with all resource extensions, but it is very handy to look things up. I can remove all of that code and people can put the `t.Logf("%+v", extension)` if they want it...